lib/routes/pixivision/utils.ts

```diff
@@ -0,0 +1,154 @@
+import { CheerioAPI } from 'cheerio';
+import { config } from '@/config';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+
+const multiImagePrompt = {
+    en: (count) => `${count} images in total`,
+    zh: (count) => `共${count}张图`,
+    'zh-tw': (count) => `共${count}張圖`,
+    ko: (count) => `총 ${count}개의 이미지`,
+    ja: (count) => `計${count}枚の画像`,
+};
+
+interface Tweet {
+    user: {
+        name: string;
+        screen_name: string;
+    };
+    text: string;
+    photos?: Array<{ url: string }>;
+    created_at: string;
+    favorite_count: number;
+    conversation_count: number;
+}
+
+export async function processContent($: CheerioAPI, lang: string): Promise<string> {
+    // 移除作者頭像
+    $('.am__work__user-icon-container').remove();
+
+    // 插畫標題&作者
+    $('.am__work__title').attr('style', 'display: inline;');
+    $('.am__work__user-name').attr('style', 'display: inline; margin-left: 10px;');
+
+    // 處理多張圖片的提示
+    $('.mic__label').each((_, elem) => {
+        const $label = $(elem);
+        const count = $label.text();
+        const $workContainer = $label.parentsUntil('.am__work').last().parent();
+        const $titleContainer = $workContainer.find('.am__work__title-container');
+
+        $titleContainer.append(`<p style="float: right; margin: 0;">${multiImagePrompt[lang](count)}</p>`);
+        $label.remove();
+    });
+
+    // 插畫間隔
+    $('.article-item, ._feature-article-body__pixiv_illust').after('<br>');
+
+    // Remove Label & Tags
+    $('.arc__thumbnail-label').remove();
+    $('.arc__footer-container').remove();
+
+    // pixivision card
+    $('article._article-card').each((_, article) => {
+        const $article = $(article);
+
+        const $thumbnail = $article.find('._thumbnail');
+        const thumbnailStyle = $thumbnail.attr('style');
+        const bgImageMatch = thumbnailStyle?.match(/url\((.*?)\)/);
+        const imageUrl = bgImageMatch ? bgImageMatch[1] : '';
+
+        $thumbnail.remove();
+
+        if (imageUrl) {
+            $article.prepend(`<img src="${imageUrl}" alt="Article thumbnail">`);
+        }
+    });
+
+    // 處理 tweet
+    const tweetPromises = $('.fab__script')
```

Comment:
`.fab__script` are iframes. Please keep it intact instead of fetching them again.

Reply:
Some Pixivision articles use Twitter posts to display images as part of content.
Without processing these, users can't see the full article directly.
They must jump to Twitter to view the images. This breaks the flow.

Reply:
Replace them with iframes just like what the site does.
In `https://www.pixivision.net/ja/a/10114` from `/pixivision/ja/interview`, you can have something like 
```html
<iframe id="twitter-widget-0" scrolling="no" frameborder="0" allowtransparency="true" allowfullscreen="true" class="" style="position: static; visibility: visible; width: 550px; height: 804px; display: block; flex-grow: 1;" title="X Post" src="https://platform.twitter.com/embed/Tweet.html?id=1815826484340351332"></iframe>
```

Reply:
Using this iframe has two issues:
1. Height can't adjust dynamically
2. Twitter posts won't show in RSS readers that block scripts

However, the current method also has two problems:
1. Fetching data for each Twitter post slows down route response time
2. Videos can't be processed

Any suggestions?